src/Data/List/Relation/Binary/Sublist/Setoid/Properties.agda

```diff
@@ -203,6 +205,31 @@ module _ {as bs : List A} where
   reverse⁻ : reverse as ⊆ reverse bs → as ⊆ bs
   reverse⁻ = HeteroProperties.reverse⁻
 
+------------------------------------------------------------------------
+-- merge
+
+module Merge {ℓ′} (_≤_ : Rel A ℓ′) (dto : IsDecTotalOrder _≈_ _≤_)  where
```

Comment:
Making the change described below would then allow us to get rid of this named module. These names should be exported publicly from this module anyway, not hidden away in a submodule.